schemas/rum/error-schema.json

```diff
@@ -47,6 +47,40 @@
               "description": "Stacktrace of the error",
               "readOnly": false
             },
+            "causes": {
+              "type": "array",
+              "description": "Causes of the error",
+              "maxItems": 10,
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 suggestion:** ‏the generated code seems to be quite verbose due to that. I'd be in favor of not putting this limit in the schema.

Reply:
😅 wow good catch, yes I am up for removing it as a result